src/app/dataset/depublication/sortable-group/sortable-group.component.html

```diff
@@ -0,0 +1,12 @@
+<ng-template #sortableGroupTemplate>
+  <app-sortable-header
+    (onSet)="onSetHandler($event)"
+    [conf]="{
+      translateKey: item.translateKey,
+      fieldName: item.fieldName,
+      cssClass: conf.cssClass
+    }"
+    *ngFor="let item of conf.items"
```

Comment:
Would it be a good idea to rename the two different 'conf' variables that are referenced here so that their names are different?